### PR TITLE
Settings advanced panels

### PR DESCRIPTION
**Pull request by AmerAnsari**

### CODE REVIEW

**Comment by AmerAnsari** [comment]
@AmirSavand Fix textarea Disable

File: src/app/components/dash/settings/settings-advanced/settings-advanced.component.ts [hunk 1]
```diff
@@ -90,4 +174,57 @@ export class SettingsAdvancedComponent implements OnInit {
...
+   * Update google adSense
+   */
+  updateGoogleAdSense(): void {
+    this.advancedLoading = true;
+    this.blogService.updateGoogleAdsense(this.googleAdSenseForm.value).subscribe((data: BlogSettings): void => {
+      this.advancedLoading = false;
+      this.toast.info(this.translate.instant('TOAST_UPDATE'), this.translate.instant('GOOGLE_ADSENSE'));
+      this.settings = data;
+      this.googleAdSenseErrors = {};
+    }, (error): void  => {
```
Comment (Alir3z4) [inline src/app/components/dash/settings/settings-advanced/settings-advanced.component.ts]: `errors` needs to have type here.

File: src/app/components/dash/settings/settings-advanced/settings-advanced.component.ts [hunk 2]
```diff
@@ -90,4 +174,57 @@ export class SettingsAdvancedComponent implements OnInit {
...
+    this.advancedLoading = true;
+    this.blogService.updateRemoveBranding(this.removeBrandingForm.value.remove_branding)
+      .subscribe((data: BlogSettings): void => {
+      this.advancedLoading = false;
+      this.toast.info(this.translate.instant('TOAST_UPDATE'),
+        this.translate.instant('REMOVE_GONEVIS_BRANDING'));
+      this.settings = data;
+    });
+    this.blogService.updateFooterText(this.removeBrandingForm.value.set_footer_text)
+      .subscribe((data: BlogSettings): void => {
```
Comment (Alir3z4) [inline src/app/components/dash/settings/settings-advanced/settings-advanced.component.ts]: I see only success response has been handled, how about error/failure response ?

File: src/app/components/dash/settings/settings-advanced/settings-advanced.component.ts [hunk 3]
```diff
@@ -90,4 +174,57 @@ export class SettingsAdvancedComponent implements OnInit {
...
     });
   }
+
+  /**
+   * Update remove branding
+   */
+  updateRemoveBranding(): void {
+    this.advancedLoading = true;
+    this.blogService.updateRemoveBranding(this.removeBrandingForm.value.remove_branding)
+      .subscribe((data: BlogSettings): void => {
```
Comment (Alir3z4) [inline src/app/components/dash/settings/settings-advanced/settings-advanced.component.ts]: I see only success response has been handled, how about error/failure response ?

**Comment by ArsalanSavand** [comment]
Done. Review required :)

**Pull request merged**